addons/rating/tests/test_controller.py

```diff
@@ -23,3 +31,15 @@ def test_controller_rating(self):
         url = '/rate/%s/submit_feedback' % access_token
         req = self.url_open(url)
         self.assertEqual(req.status_code, 200, "Response should = OK")
+
+        # Test that partner can give the rating.
+        url = '/rate/%s/5' % access_token
+        res = self.url_open(url)
+        self.assertEqual(res.status_code, 200)
+        self.assertTrue(rating_test.consumed)
+
+        # Test that internal user are not cannot give the rating
```

Comment:
"Test that if you access the rating URL as logged in and not the assigned customer, you cannot rate".